src/components/ReadingDots.jsx

```diff
@@ -43,21 +44,16 @@ const ReadingDots = ({ actors, currentActor, currentScript }) => {
       // this helps reduce many small changes into one larger change request
       clearTimeout(scrollTimeout);
 
-      scrollTimeout = setTimeout(() => {
+      scrollTimeout = setTimeout(async () => {
         let readingDotsHeightOffset =
           document.getElementsByTagName('body')[0].getBoundingClientRect().height -
           document.getElementById('readingDots').getBoundingClientRect().height;
 
         let position = (window.scrollY - readingDotsHeightOffset) / (document.body.scrollHeight - readingDotsHeightOffset);
         position = position.toPrecision(4);
 
-        fetch('/api/positions', {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-          },
-          body: JSON.stringify({actorId:currentActor.id, scriptId: currentScript, position: position})
-        });
+          // fetch request to update position
+          await postPosition(currentActor.id, currentScript, position)
```

Comment:
Could you run the code formatter? The alignment here doesn't match up with the other lines